src/spid-sdk.js

```diff
@@ -18,22 +19,26 @@ function globalExport(global) {
 }
 
 function init(opts, callback) {
+    callback = util.makeAsync(callback || noop);
     config.init(opts);
     if(!config.options().noGlobalExport) {
         globalExport.call(this, window);
     }
     _initiated = true;
-    if(callback) {
-        callback();
-    }
+     callback();
 }
 
 function version() {
     return _version;
 }
 
 function hasSession(callback) {
-    callback = callback || noop;
+    var key = 'session';
+    callback = util.makeAsync(callback || noop);
```

Comment:
Would you be kind to move the `noop` to `spid-util.js`? Not a hard requirement though.

Reply:
👍

Reply:
Hmm, it does not really provide any value in moving it. Moving it means more characters though, which means a few bytes bigger sdk :)